server/chat-commands/info.ts

```diff
@@ -2543,6 +2546,7 @@ export const commands: Chat.ChatCommands = {
 		room.sendRankedUsers(message, '%');
 		room.sendMods(
 			Utils.html`|uhtml|request-${user.id}|<div class="infobox">${user.name} wants to show <a href="${link}">${link}</a><br>` +
+			(comment ? Utils.html`${comment}<br>` : '') +
```

Comment:
```suggestion
			(comment ? Utils.html`Comment: ${comment}<br>` : '') +
```
Just to make it clearer.